src/drivers/general/queries/MultithreadedAVCondition/MultithreadedAVCondition.qhelp

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+	<overview>
+		<p>
+			This warning indicates that a thread has potential to access deleted objects if preempted. This query does not currently verify the use of any synchronization mechanisms, so it may produce false positives.
```

Comment:
After reviewing the code for the original Code Analysis version of this rule, it looks like that version is specific to checking cases where InterlockedDecrement is used.  We should update this text to be very explicit about that difference - maybe just add "Unlike the Code Analysis version of this query" ahead of the second sentence.